dask_cuda/tests/test_explicit_comms.py

```diff
@@ -20,7 +20,7 @@
 from dask_cuda.utils import get_ucx_config
 
 pytestmark = pytest.mark.skipif(
-    sys.version_info.minor < 80,
+    sys.version_info.minor < 8,
```

Comment:
It seems this is still needed because of https://github.com/dask/distributed/pull/5380, when that gets merged will we be able to remove this `skipif`?

Reply:
Yes, it is still needed. It is just fixing the typo -- we want to run the test on Python v3.8+ **not** Python v3.80+ :P

Reply:
Yes, that makes sense. Will https://github.com/dask/distributed/pull/5380 resolve that, i.e., will we be able to run on Python 3.7 again after it's merged?

Reply:
https://github.com/dask/distributed/pull/5380 is in now, are we safe removing the `skipif` here?

Reply:
> [dask/distributed#5380](https://github.com/dask/distributed/pull/5380) is in now, are we safe removing the `skipif` here?

True, but let's merge to get the type hints and then merge https://github.com/rapidsai/dask-cuda/pull/754